yt_dlp/extractor/youtube.py

```diff
@@ -1081,7 +1085,7 @@ def _extract_video(self, renderer):
             'uploader_url': format_field(channel_handle, None, 'https://www.youtube.com/%s', default=None),
             'thumbnails': self._extract_thumbnails(renderer, 'thumbnail'),
             'timestamp': (self._parse_time_text(time_text)
-                          if self._configuration_arg('approximate_date', ie_key=YoutubeTabIE)
+                          if self._configuration_arg('approximate_metadata', ie_key=YoutubeTabIE)
```

Comment:
keep `approximate_date` too for compat (don't document) and raise `self._downloader.deprecated_feature` if it is used